### PR TITLE
dd-34-hot-fix--update-package-lock

### PR DESCRIPTION
**Pull request by RykovaMariia**
# Task

[update package-lock](https://divas801801.atlassian.net/jira/software/projects/DD/boards/2?selectedIssue=DD-34)

# Description

update builder, planess/train-a-backend and package-lock for fix ci

## Pull request type

Please check the type of change your PR introduces:

- [ ] 🍕 Feature
- [x] 🐛 Bugfix
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update
- [ ] 🤖 Build related changes
- [ ] 🔥 Performance Improvements
- [ ] Other (please describe):

# Demo (situationally)

# Has It Been Tested?

- [ ] Yes
- [ ] No, and this is why: please replace this line with details on why tests have not been included
- [x] Tests are not needed for this task
- [ ] I need help with writing tests

### CODE REVIEW

**Review by Dewlar — APPROVED** [review]

File: src/env.d.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+// // You can modify the name of the variable in angular.json.
+// // ngxEnv: {
+// //  define: '_NGX_ENV_',
+// // }
+// declare const _NGX_ENV_: Env;
+
+// // 3. Use process.env.YOUR_ENV_VAR in your code. (deprecated)
+// declare namespace NodeJS {
+//   export interface ProcessEnv extends Env {}
+// }
```
Comment (TELEUZI) [inline src/env.d.ts]: Why commented?

Comment (RykovaMariia) [reply src/env.d.ts]: Karina will add this later, no one has used it yet

**Review by ChickenMcNuggetss — APPROVED** [review]

File: src/env.d.ts (hunk 1 above)
Comment (TELEUZI) [reply src/env.d.ts]: Why add file then?

Comment (RykovaMariia) [reply src/env.d.ts]: it is automatically added, now I will try to delete it

Comment (RykovaMariia) [reply src/env.d.ts]: I deleted it, thanks

**Pull request merged**